demos/basics/loading-saving-neo4j.ipynb

```diff
@@ -25,10 +25,14 @@
    "cell_type": "markdown",
```

Comment:
<blockquote>StellarGraph also have experimental support for&nbsp;<a href="https://github.com/stellargraph/stellargraph/raw/803b38b9ed9ca846b6868ae0c6301bd160bc84fc/demos/connector/neo4j/README.md" target="_blank">running some algorithms directly using Neo4j</a>.</blockquote><p>"has" instead of "have"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1652/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>